docs/attributes-registry/network.md

```diff
@@ -0,0 +1,131 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
```

Comment:
do we need to have a `linkTitle` here?

Reply:
When the `linkTitle` is not provided, the MD title is the default (in this case `Network`), which is what we want in this case.